web/themes/custom/novel/templates/components/hero.html.twig

```diff
@@ -6,9 +6,9 @@
 %}
 
 <section {{ attributes.addClass(classes) }}>
-  {% if link %}
-    {% set url = link.0['#url'] %}
-    {% set target = url.options.target|default('_self') %}
+  {% if link.0 %}
+    {% set url = link.0['#url'].external ? link.0['#url'].uri : link.0['#url'].toString() %}
+    {% set target = content.field_banner_link[0]['#url'].options.target|default('_self') %}
```

Comment:
Why is target taken from `content.field_banner_link` when the URL is taken from `link`? How can we be sure that it's actually the same link?

It looks like `hero.html.twig` is included from `paragraphs/paragraph--hero.html.twig`, `components/full-event.html.twig` and `layout/node--branch--full.html.twig`, neither of which seems to pass either the `link` or `content` variables, and I can't find a preprocess function for it either. What's going on?

Reply:
This is a copy-paste mistake, sorry 😅 Thanks for finding it.